pages/common/tldr.md

```diff
@@ -27,3 +27,6 @@
 - [l]ist all pages for the current platform and `common`:
 
 `tldr --list`
+
+- [l]ist all available subcommands for the current platform in tldr pages:
```

Comment:
```suggestion
- [l]ist all available subcommand pages for a command:
```

Reply:
Why you resolved this suggestion?

Reply:
yesterday I skimmed and mistook resolving it